lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1244,6 +1214,20 @@ public function prepare_item_for_response( $post, $request ) {
 	}
 
 	/**
+	 * Overwrite the default protected title format.
+	 *
+	 * By default WordPress will show password protected posts with a title of
+	 * "Protected: %s", as the REST API communicates the protected status of a post
+	 * in a machine readable format, we remove the "Protected: " prefix.
```

Comment:
Sure can do - I was getting at there's no need to mark a post as "Protected: $x" because we communicate that it's protected in other ways.